apps/vs-code-designer/src/app/commands/workflows/switchToDotnetProject.ts

```diff
@@ -254,6 +255,10 @@ async function getArtifactNamesFromProject(target: vscode.Uri): Promise<Record<s
       continue;
     }
 
+    if (file === parametersFileName) {
+      artifactDict['connections'].push(parametersFileName);
```

Comment:
I didn't find any test coverage for this, am I missing something?

Reply:
No, I think there are no tests for this file.